scenarios/kubevirt/kubevirt-vm-outage.yaml

```diff
@@ -0,0 +1,7 @@
+scenarios:
+  - name: "kubevirt outage test"
+    scenario: kubevirt_vm_outage
```

Comment:
Even if you plan to have multiple `kubevirt_*` sub-scenario , that I understand you might have seen from other scenarios, it's better to do not write code that [you ain't gonna need it](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). This behavior is already mapped in the API itself through the `get_scenario_types()` and you implemented it correctly,